src/main/java/io/reactivex/plugins/RxJavaPlugins.java

```diff
@@ -186,10 +187,10 @@ public static boolean isLockdown() {
      * @param defaultScheduler the hook's input value
      * @return the value returned by the hook
      */
-    public static Scheduler initComputationScheduler(Scheduler defaultScheduler) {
+    public static Scheduler initComputationScheduler(Callable<Scheduler> defaultScheduler) {
```

Comment:
Null should not be allowed as a return value from the `Callable` nor from the init `Function`.